README.md

```diff
@@ -96,6 +100,32 @@ Setting up the nvidia support should be automatic the hardware is present, but y
 snap set docker nvidia-support.disabled=true
 ```
 
+### Usage examples
+
+Generic example usage would look something like:
+
+```shell
+docker run --rm --runtime nvidia --gpus all {cuda-container-image-name}
+```
+
+or
+
+```shell
+docker run --rm --runtime nvidia --env NVIDIA_VISIBLE_DEVICES=all {cuda-container-image-name}
+```
+
+If your container image already has appropriate environment variables set, may be able to just specify the nvidia runtime with no additional args required.
+
+Please refer to [this guide](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/docker-specialized.html) for mode detail regarding environment variables that can be used.
+
+*NOTE*: library path and discovery is automatically handled, but binary paths are not, so if you wish to test using something like the `nvidia-smi` binary passed into the container from the host, you could can either specify the full path of set the PATH environment variable.
```

Comment:
Correct me if I am wrong, but I believe the last sentence should be:

"you COULD either specify the full path OR set the PATH environment variable"

Removing the "can" and replacing "of" with "or". Is my assumption correct?

Reply:
Well spotted, and thank you :+1: 

Updated.